fastlane/fastlane
# fix the loading of certain passwords from the keychain (fixes #21817)

If a password contains certain characters (such as a backslash), the security command line tool will output the password in hexadecimal format followed by a quoted escaped string. Version 0.1.5 of the security gem supports this format, so updating to this version will make it so _fastlane_ can properly load saved passwords that contain these special characters.

### Checklist

  - (Unit tests for this issue already exist in the fastlane-community/security repo.)

### Motivation and Context

Resolves #21817

### Description

Updates the security gem to the latest version, from version 0.1.3 to 0.1.5.

I tested these changes by installing this version of fastlane, and then running `fastlane sigh download_all` after saving my account credentials to the keychain.

### Testing Steps

1. Run `fastlane fastlane-credentials add` and provide a password that contains a `\`
2. Run `fastlane sigh download_all` and observe that the previously saved password is used for the login
